### DISCUSSION TITLE
Hi guys. I am looking for trackball sensor pcb.

### DISCUSSION THREAD

**Discussion by NuuuDle13** [Yowkees/keyball#618] [Q&A]
Maybe someone have gerber for the trackball sensor pcb. Please share it. 🙏🙏🙏

**Comment by mkbkakwk** [comment]
The PCB files are in [keyball46](https://github.com/Yowkees/keyball/tree/main/keyball46/ball_design_data). However, there are issues with the provided Gerber files - there are no holes drilled and no solder pads, which seems intentional.
